tutorials/klusterai-api/.pages

```diff
@@ -10,4 +10,7 @@ nav:
   - 'Fine-tuning': finetuning-sent-analysis.ipynb
   - 'Image analysis': image-analysis.ipynb
   - 'Effective prompt engineering': effective-prompt-engineering.ipynb
-  - 'Uploading large files': 'uploads-api.ipynb'
\ No newline at end of file
+  - 'Uploading large files': 'uploads-api.ipynb'
+  - 'Integrating Tools': 'integrating-tools.ipynb'
+  - 'RAG': 'rag.ipynb'
```

Comment:
Is there anything else we can add to the title that would provide more context? Not sure if there will be more than one RAG tutorial at some point?